lib/ansible_tower_client/api.rb

```diff
@@ -4,6 +4,8 @@ module AnsibleTowerClient
   class Api < Connection
     include Logging
 
+    DEFAULT_ERROR_MSG = "An unknown error was returned from the provider"
```

Comment:
I don't know if it's necessary - I generally like to see default text in a constant for readability.

I'll freeze it.  Thanks!